### PR TITLE
Remove FixedBackendResolver

### PR DESCRIPTION
**Pull request by Martin2112**
This was previously used in situations where a comma separated list of
backends was provided in the `rpcBackend` flag rather than a single value.

It has been replaced by equivalent functionality using a newer gRPC API.
However this support was only intended for use in integration tests. In
production we recommend the use of etcd or a gRPC load balancer.

### Checklist

- [X] I have updated the [CHANGELOG](CHANGELOG.md).
  - Adjust the draft version number according to [semantic versioning](https://semver.org/) rules.
- [ ] I have updated [documentation](docs/) accordingly.

### CODE REVIEW

File: trillian/ctfe/ct_server/main.go [hunk 1]
```diff
@@ -140,10 +141,19 @@ func main() {
 			etcdRes.Update(ctx, *etcdMetricsService, byeMetrics)
 		}()
 	} else if strings.Contains(*rpcBackend, ",") {
-		glog.Infof("Using FixedBackendResolver")
-		// Use a fixed endpoint resolution that just returns the addresses configured on the command line.
-		res := util.FixedBackendResolver{}
-		dialOpts = append(dialOpts, grpc.WithBalancer(grpc.RoundRobin(res)))
+		// This should probably not be used in production. Either use etcd or a gRPC
+		// load balancer. It's used by the integration tests.
+		glog.Warningf("Multiple RPC backends from flags not recommended for production.")
```
Comment (AlCutter) [inline trillian/ctfe/ct_server/main.go:146]: Seems that you don't need the `f` variant since there are no params?

Comment (AlCutter) [reply trillian/ctfe/ct_server/main.go:146]: Might also be worth adding the recommendation text from the comment to the logged warning too - save folks from having to dig in the code to know what to do instead?

Comment (Martin2112) [reply trillian/ctfe/ct_server/main.go:146]: Removed the 'f' and expanded the comment.

**Comment by Martin2112** [comment]
PTAL.

**Pull request merged**